src/PSRule.Rules.Azure/rules/Azure.Databricks.Rule.yaml

```
@@ -25,3 +25,43 @@ spec:
         isBoolean: true
       - field: properties.parameters.enableNoPublicIp.value
         equals: true
+
+---
+# Synopsis: Use non-Trial SKUs for Databricks workspaces.
+apiVersion: github.com/microsoft/PSRule/v1
+kind: Rule
+metadata:
+  name: Azure.Databricks.SKU
+  ref: AZR-000409
+  tags:
+    release: GA
+    ruleSet: 2024_01
+    Azure.WAF/pillar: Performance Efficiency
+spec:
+  type:
+  - Microsoft.Databricks/workspaces
+  condition:
+    allOf:
+      - field: sku.name
+        notEquals: trial
```

Don't have to change it. More just a call out that this is a simpler alternative:

```suggestion
  condition:
     field: sku.name
     notEquals: trial
```